metals/src/main/scala/scala/meta/internal/builds/Digest.scala

```diff
@@ -90,16 +90,40 @@ object Digest {
       Set("gradle", "groovy", "gradle.kts", "java", "kts").exists(
         path.toString().endsWith(_)
       )
+    val isXml = ext == "xml"
 
     if (isScala && path.isFile) {
       digestScala(path, digest)
     } else if (isGradle && path.isFile) {
       digestGeneralJvm(path, digest)
+    } else if (isXml){
+      digestXml(path, digest)
     } else {
       true
     }
   }
 
+  def digestXml(
+      file: AbsolutePath,
+      digest: MessageDigest
+  ): Boolean = {
+    try {
+      Files
```

Comment:
FYI, not sure we want to do it, but one option is to parse the file with `scala.xml.XML.loadFile` and digest the parsed XML

Reply:
Added, I think I was just a bit lazy previously and forgot about it.